emulator/src/components/hdw-bzr/hdw-bzr.c

```diff
@@ -488,3 +488,42 @@ void bzrResume(void)
         }
     }
 }
+
+/**
+ * @brief Save the state of the buzzer so that it can be restored later, perhaps
+ * after playing a different sound.
+ *
+ * @return A void-pointer which can be passed back to bzrRestore()
+ */
+void* bzrSave(void)
```

Comment:
Next year I have to do a better job pulling common buzzer code out of `components`.